activerecord/lib/active_record/relation/query_methods.rb

```diff
@@ -512,13 +512,12 @@ def order!(*args) # :nodoc:
       self
     end
 
-    # Allows to specify an order by a specific set of values. Depending on your
-    # adapter this will either use a CASE statement or a built-in function.
+    # Allows to specify an order by a specific set of values.
     #
     #   User.in_order_of(:id, [1, 5, 3])
     #   # SELECT "users".* FROM "users"
-    #   #   ORDER BY FIELD("users"."id", 1, 5, 3)
     #   #   WHERE "users"."id" IN (1, 5, 3)
+    #   #   ORDER BY CASE WHEN "users"."id" = 1 THEN 1 WHEN "users"."id" = 5 THEN 2 WHEN "users"."id" = 3 THEN 3 END ASC
```

Comment:
This ends up being a bit long when rendered:

![apidocs-in_order_of](https://user-images.githubusercontent.com/771968/201408848-db677145-7a7b-401c-aa01-d36085d180a3.png)

What do you think about breaking it up?

```suggestion
    #   #   ORDER BY CASE
    #   #     WHEN "users"."id" = 1 THEN 1
    #   #     WHEN "users"."id" = 5 THEN 2
    #   #     WHEN "users"."id" = 3 THEN 3
    #   #   END ASC
```

Reply:
agree @jonathanhefner I'll update.